Makefile

```diff
@@ -101,21 +101,18 @@ test-update: update-prepare update
 		exit 1; \
 	} \
 
-test-e2e: image hyperkube
-	docker push $(IMAGE)
-	docker build -t $(TEST_IMAGE) \
-		--build-arg K8S_VERSION=$(K8S_VERSION) \
-		--build-arg AKSENGINE_VERSION=$(AKSENGINE_VERSION) \
-		tests/k8s-azure
-	docker run --env-file $(K8S_AZURE_ACCOUNT_CONFIG) \
-		-e K8S_AZURE_TEST_ARTIFACTS_DIR=$(ARTIFACTS) \
-		-v $(WORKSPACE):$(WORKSPACE) \
-		-v $(ARTIFACTS):$(ARTIFACTS) \
-		$(TEST_IMAGE) e2e -v -caccm_image=$(IMAGE) \
-		-ctype=$(SUITE) \
-		-csubject=$(SUBJECT) \
-		-cmanifest=$(TEST_MANIFEST) \
-		-chyperkube_image=$(HYPERKUBE_IMAGE)
+test-e2e: deploy
```

Comment:
Please delete deploy target. We should support running the e2e for existing clusters

Reply:
Oksy Sir.....I thought like we should make a cluster before running tests . Okay. But should we check for existence of clusters?Whether that cluster is up and running?